extensions/README.md

```diff
@@ -47,22 +47,48 @@ types. This is done via three distinct extension mechanisms:
 
 Map extensions allow extending CoRIM maps with additional keys, effectively
 defining new fields for the corresponding structures. In the code base, these
-can be identified by the embedded `Extensions` struct. These are
-
-- `comid.Comid`
-- `comid.Entity`
-- `comid.FlagsMap`
-- `comid.Mval`
-- `comid.Triples`
-- `corim.Entity`
-- `corim.Signer`
-- `corim.UnsignedCorim`
+can be identified by the embedded `Extensions` struct. Each extensible type has
```

Comment:
Just a quick question: Line 2 of the README.md refer to V2 of the specification, are you adhereing to V4 or still on V2?

Reply:
I'll update it to 04, but technically, the current implementation doesn't really fully adhere to any particular rev. E.g. we're only now fixing the profile claim in this pull request, and that was incorrect since rev. 00.

Reply:
ok, thanks, this was just a quick check!